node/src/main.rs

```diff
@@ -1,58 +1,36 @@
 use clap::Parser as _;
-use ethereum::chain::{
-    EthereumAdapterSelector, EthereumBlockRefetcher, EthereumRuntimeAdapterBuilder,
-    EthereumStreamBuilder,
-};
-use ethereum::{BlockIngestor, EthereumNetworks};
```

Comment:
Nice! I like that this tightens up `main` overall - it had way too much control flow mixed with intricate but somewhat boring detail.